Resources/Prototypes/Palettes/departmental.yml

```diff
@@ -10,5 +10,6 @@
     bar: "#79150096"
     epistemics: "#D381C996" # DeltaV - Epistemics Department replacing Science
     logistics: "#A4610696" # DeltaV - Logistics Department replacing Cargo
+    legal: "#6b2833" #DeltaV - Added Legal (let me know if this needs to be moved. just figured it's easier to put it here.)
```

Comment:
Justice? But yes, leave it here.

Reply:
It should say justice if we're going with that department name